integrationTests/vm/txDataBuilder.go

```diff
@@ -0,0 +1,127 @@
+package vm
```

Comment:
This is a very useful tool that can be used in other packages other than integrationTests. 
Suggestion to move in /testscommon/txData and call this struct Builder.

Then we would have txData.NewBuilder().....

Reply:
Moved to package `txDataBuilder`, not `txData`. This avoids ambiguity when we use `txData` as a variable name, which is preferred.